pause.go

```diff
@@ -19,18 +22,17 @@ Use runc list to identify instances of containers and their current status.`,
 		if err := checkArgs(context, 1, exactArgs); err != nil {
 			return err
 		}
-		rootlessCg, err := shouldUseRootlessCgroupManager(context)
+		container, err := getContainer(context)
 		if err != nil {
 			return err
 		}
-		if rootlessCg {
-			logrus.Warnf("runc pause may fail if you don't have the full access to cgroups")
-		}
-		container, err := getContainer(context)
-		if err != nil {
+		if err := container.Pause(); err != nil {
+			if errors.Is(err, fs.ErrPermission) {
+				logrus.Warnf("runc pause failure might be caused may fail if you don't have the full access to cgroups")
```

Comment:
Maybe should changed to:
"runc pause failed: maybe because you don't have the full access to cgroups"

Reply:
Sorry, I forgot to finish the message :) 

Fixed.